Formula/emacs-plus.rb

```diff
@@ -107,11 +107,17 @@ class EmacsPlus < Formula
   depends_on "dbus" => :optional
   depends_on "gnutls" => :recommended
   depends_on "librsvg" => :recommended
-  # Emacs does not support ImageMagick 7:
+  # Emacs 26.x does not support ImageMagick 7:
   # Reported on 2017-03-04: https://debbugs.gnu.org/cgi/bugreport.cgi?bug=25967
   depends_on "imagemagick@6" => :recommended
+  # Emacs 27.x (current HEAD) does support ImageMagick 7:
+  # Prefer ImageMagick 7 when both on
+  depends_on "imagemagick@7" => :optional
+
   depends_on "mailutils" => :optional
 
+  depends_on "jansson" => :optional
```

Comment:
Once 27 is out this one should become recommended!

Reply:
> Once 27 is out this one should become recommended!

It seems that the native json support is on with default.

 https://github.com/emacs-mirror/emacs/blob/eb6bbd9fb175cacdfdc54c1187f5785ed3858f2f/configure.ac#L432